src/main/java/com/getindata/connectors/http/sink/httpclient/JavaNetHttpClient.java

```diff
@@ -0,0 +1,110 @@
+package com.getindata.connectors.http.sink.httpclient;
+
+import com.getindata.connectors.http.HttpClientResponse;
+import com.getindata.connectors.http.sink.HttpSink;
+import com.getindata.connectors.http.sink.HttpSinkRequestEntry;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+/**
+ * An implementation of {@link com.getindata.connectors.http.HttpClient} that
+ * uses Java 11's {@link HttpClient}.
+ */
+public class JavaNetHttpClient implements com.getindata.connectors.http.HttpClient {
+  private final HttpClient httpClient;
+
```

Comment:
This is what I was talking about in https://github.com/getindata/flink-http-connector/pull/4/files/fa3b89de61a214333aebeaa5311fd24987f07b4a..f3c496eaea73fac2486711607f97727097249a25#r904074482

Please change interface name to `SinkHttpClient` so you wouldnot have to use full name with apckage to impleent an interface.